modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/info/collection/provider/SXPBlueprintInfoCollectionProvider.java

```diff
@@ -61,7 +61,6 @@
  * @author Tibor Lipusz
  */
 @Component(
-	enabled = false,
```

Comment:
We need `enabled = false` because SXP components are part of EE activation policy, see details in https://liferay.atlassian.net/l/cp/rQ2Ds8jJ